src/nupic/data/aggregator.py

```diff
@@ -20,14 +20,14 @@
 # http://numenta.org/licenses/
 # ----------------------------------------------------------------------
 
-import os
-import time
-import datetime
 from collections import defaultdict
-
+import datetime
```

Comment:
Here, I simply rearranged the imports to be in sorted order (unrelated to the core of the change)